app/config/services/services.xml

```diff
@@ -213,7 +213,10 @@
         <service id="AppBundle\Normalizer\IdeasWorkshopNormalizer">
             <argument type="service" id="api_platform.serializer.normalizer.item" />
         </service>
-        <service id="AppBundle\Normalizer\AuthorNormalizer">
+        <service id="AppBundle\Normalizer\IdeaDenormalizer">
+            <argument type="service" id="AppBundle\Normalizer\AuthorDenormalizer" />
```

Comment:
The custom `AuthorDenormalizer` is really needed here ?

Reply:
Changed.